packages/core/src/App/Containers/RealAccountSignup/__tests__/account-wizard.spec.tsx

```diff
@@ -3,6 +3,7 @@ import { WS } from '@deriv/shared';
 import { render, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import AccountWizard from '../account-wizard';
+import { mockStore, StoreProvider } from '@deriv/stores';
```

Comment:
why do we need this? The component is not using useStores and still use connect method

shouldnt we mock the hook insted?

Reply:
Yes we do, we're using a hook which is based on useStore and we shouldn't mock the hook instead pass the value to the provider & connect method will be removed as well

Reply:
@shahzaib-deriv i dont think so. In our codebase, we have spec files with mocking the hooks. . So here since the component is still using connect method and only hooks is using stores, i dont think we need to wrap the component with StoreProvider. 
Maybe you can refer packages/cashier/src/pages/withdrawal/__tests__/withdrawal.spec.tsx